package.nls.json

```diff
@@ -780,7 +780,7 @@
         "comment": ["{Locked='python -m jupyter notebook --version'}"]
     },
     "DataScience.savePngTitle": "Save Image",
-    "DataScience.jupyterSelectURIQuickPickTitle": "Pick how to connect to Jupyter",
+    "DataScience.jupyterSelectURIQuickPickTitle": "Enter the URI of the running Jupyter server",
```

Comment:
Isn't there a localize.ts entry as well?

Reply:
The bug here was that this file didn't have the new text. the localize.ts file alredy says 'Enter the URI of the running Jupyter server'. 

On web we use the nls.json file. On desktop the localize.ts file is translated by the translation files.

Reply:
@rchiodo Ahh should have paid more attention to the bug. I just saw one file and not the other. Sounds good.